src/kernel/mconfig.ml

```diff
@@ -266,10 +272,13 @@ let merge_merlin_config dot merlin ~failures ~config_path =
     suffixes = dot.suffixes @ merlin.suffixes;
     stdlib = (if dot.stdlib = None then merlin.stdlib else dot.stdlib);
     unit_name = (if dot.unit_name = None then merlin.unit_name else dot.unit_name);
-    wrapping_prefix =
```

Comment:
Why is `wrapping_prefix` being deleted here?  Probably this belongs  in a future PR that deletes it entirely?